packages/augur-ui/src/modules/events/actions/add-update-transaction.ts

```diff
@@ -44,8 +44,7 @@ const ADD_PENDING_QUEUE_METHOD_CALLS = [
   MIGRATE_FROM_LEG_REP_TOKEN,
   REDEEMSTAKE,
   BATCHCANCELORDERS,
-  TRADINGPROCEEDSCLAIMED,
-  CLAIMMARKETSPROCEEDS
```

Comment:
removing `CLAIMMARKETSPROCEEDS` here does Notification `view` button show processing if user clicks `claim all` ?

Reply:
it does. we only want it if multiple markets have been claimed so had to remove it there

Reply:
I think this can be left in. It should not conflict with individual claims. the notification button will just show processing while the individual claim is being processed. it will disable the notification button though.

Reply:
and then they can't get back to the modal to make more claims if they are in the process with one?

Reply:
could have the notification button not disable.

Reply:
just another param to pass into processingButton

Reply:
its debatable